packages/pwa-kit-create-app/assets/pwa/pwa-kit.config.json

```diff
@@ -0,0 +1,25 @@
+{
+    "app": {
+        "defaultSiteId": "RefArchGlobal",
```

Comment:
@kevinxh from what we discussed just now, we would want to have the site id to be `RefArch`.